core/common/src/ContextRealityModel.ts

```diff
@@ -22,10 +22,82 @@ export interface OrbitGtBlobProps {
   accountName: string;
 }
 
+/** Identify the Reality Data service provider
+ * @alpha
+ */
+export enum RealityDataProvider {
+  /**
+   * This is the legacy mode where the access to the 3d tiles is harcoded in ContextRealityModelProps.tilesetUrl property.
+   * It was use to support RealityMesh3DTiles, Terrain3DTiles, Cesium3DTiles
+   * You should use other mode when possible
+   * @see [[RealityDataSource.createRealityDataSourceKeyFromUrl]] that will try to detect provider from an URL
+   * */
+  TilesetUrl = "TilesetUrl",
+  /**
+   * Will provide access url from realityDataId and iTwinId on contextShare for 3dTile storage format or  OPC storage format
+   * This provider support all type of 3dTile storage fomat and OrbitPointCloud: RealityMesh3DTiles, Terrain3DTiles, Cesium3DTiles, OPC
+   * @see [[RealityDataFormat]].
+  */
+  ContextShare = "ContextShare",
+  /**
+   * Will provide Open Street Map Building (OSM) from Cesium Ion (in 3dTile format)
+  */
+  CesiumIonAsset = "CesiumIonAsset",
+}
+
+/** Identify the Reality Data storage format
+ * @alpha
+ */
+export enum RealityDataFormat {
+  /**
+   * 3dTile supported formats; RealityMesh3DTiles, Terrain3DTiles, Cesium3DTiles
+   * */
+  ThreeDTile = "ThreeDTile",
+  /**
+   * Orbit Point Cloud (OPC) storage format (RealityDataType.OPC)
+  */
+  OPC = "OPC",
+}
+
+/**
+ * Key used by ContextShare RealityDataProvider
```

Comment:
If it's only used by RealityDataProvider.ContextShare, the name should make that clear.
(Also there was just a huge pedantic PR that changed all usage of "context" and "project" - probably shouldn't be adding new names with "context" in them).

Reply:
@pmconne The official name for the service is not Reality Data Service but actually it is "ProjectWise Context Share" thus the 'context' here. It has nothing to do with what context used previously.